### DISCUSSION TITLE
Starting Pane Consistency

### DISCUSSION THREAD

**Discussion by meteorquake** [WinMerge/winmerge#1598] [Ideas]
Pretty minor matter as things go.
After a Ctrl-N I usually like to start my first paste on the left hand side (probably a standard western preference).
Currently it's a bit erratic regarding which pane starts with the focus, sometimes it starts on the left, sometimes the right, I'm not sure of the reason for the existing variability.
Cheers, David

**Comment by sdottaka** [comment]
WinMerge remembers which pane had focus when closing a window.
And it gives focus to the pane remembered after pressing Ctrl-N.

**Reply by meteorquake** [reply]
Thanks - I had the impression it was something like that; I find it rather disturbing to work-flow!
Is there a practical need for having it remember the pane in that way?
David

**Reply by sdottaka** [reply]
In older versions, when files were opened, the focus was always on the left pane. However, I often edit on the right pane, so I wanted the focus to be on the right pane at first. However, some people may prefer to edit on the left pane, so we made it so that the focus is on the pane where the focus was last time when the files are opened. However, for a new file comparison with Ctrl+N, it may be better to always have the focus on the left side.

**Comment by meteorquake** [comment]
I think using the pane which had focus when closed isn't going to help as that's going to be fairly random.
I'm sure the answer is, whenever Winmerge opens a new blank comparison it should note which of the panes is first altered, and use that as the starting pane for next time. People will likely be very consistent with their choice of left or right for where they start pasting in...
d

**Comment by sdottaka** [comment]
I agree with what you're describing, but it's a bit difficult to implement, so I've modified it to always focus on the left pane when pressing Ctrl+N. (c6e5ad5)
